tests/backup/backup_basic_test.go

```diff
@@ -153,7 +153,7 @@ var _ = BeforeSuite(func() {
 			CreateBucket(provider, globalGCPBucketName)
 			log.Infof("Bucket created with name - %s", globalGCPBucketName)
 		case drivers.ProviderNfs:
-			globalNFSBucketName = fmt.Sprintf("%s-%s", globalNFSBucketPrefix, bucketNameSuffix)
+			globalNFSBucketName = fmt.Sprintf("%s", globalNFSBucketPrefix)
```

Comment:
The bucketNameSuffix for the NFS provider is an empty string. As a result, the backup location name will have a redundant '-'.